modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -578,6 +578,7 @@
   "scim2.endpoints.users_endpoint": "$ref{server.base_path}/wso2/scim2/Users",
   "scim2.endpoints.groups_endpoint": "$ref{server.base_path}/wso2/scim2/Groups",
   "scim2.enable_complex_multivalued_attribute_support": false,
+  "scim2.enable_enhance_filtering" : false,
```

Comment:
Shouldn't this be true for new releases?
And let's have the name 'enable_filtering_enhancements' or something similar. (Avoid double verbs)